Mathlib/Topology/Category/CompHaus/EffectiveEpi.lean

```diff
@@ -223,38 +129,13 @@ theorem effectiveEpiFamily_tfae
     simp only [Discrete.functor_obj, colimit.ι_desc, Cofan.mk_pt, Cofan.mk_ι_app,
       colimit.comp_coconePointUniqueUpToIso_hom_assoc]
     ext; rfl
-  tfae_have 3 → 1
-  · apply effectiveEpiFamily_of_jointly_surjective
   tfae_finish
 
-instance precoherent : Precoherent CompHaus.{u} := by
```

Comment:
The instance for `Preregular` moved up, but where did this instance go?

Reply:
`Precoherent` now follows from `Preregular` and `FinitaryPreExtensive`. So the proof would be `inferInstance`. Then I'm not supposed to state it, right? Or maybe it should be stated as an `example`?

Reply:
I think stating it as an example is helpful. To remind people like me of this implication (-;